build/dsc/templates/dsc_type.rb.erb

```diff
@@ -55,8 +55,17 @@ Puppet::Type.newtype(:dsc_<%= resource.friendlyname.downcase %>) do
 
   @doc = %q{
     The DSC <%= resource.friendlyname%> resource type.
-    Originally generated from the following schema.mof file:
-      <%= resource.relative_mof_path %>
+    Automatically generated from
+    '<%= resource.relative_mof_path.gsub('import/dsc_resources/','') %>'
```

Comment:
I like the path used here, immediately apparent which module it comes from without the implementation details of where it was vendored in the source